release-notes/stable.yml

```diff
@@ -1,4 +1,24 @@
 releases:
+  42.20250410.3.2:
+    issues:
+      - text: "Docker Swarm DNS resolution failing in F42"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1933"
+      - text: "No network for moby containers after updating to 42.20250410.3.1"
```

Comment:
sorry I just updated the description of this :(

Reply:
updated!